addons/mrp_subcontracting/models/stock_location.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class StockLocation(models.Model):
+    _inherit = 'stock.location'
+
+    def should_bypass_reservation(self):
+        should_bypass_reservation = super(StockLocation, self).should_bypass_reservation()
+        if not should_bypass_reservation and self == self.env.company.subcontracting_location_id:
+            return True
+        return should_bypass_reservation
```

Comment:
won't work in multi company (access of two companies and subcontract amother
``` python
res = super(StockLocation, self).should_bypass_reservation()
if not should_bypass_reservation:
    self in self.env.company_ids.mapped('subcontracting_location_id)
return res
```
+ what if a company is forced to a partner and is not part of the property?

Reply:
to remove and override _action_assign, extract
```
            if move.location_id.should_bypass_reservation()\
                    or move.product_id.type == 'consu':
```
 to an helper